docs/05-engineering/accessibility.md

```diff
@@ -13,7 +13,7 @@ We implement 508 and WCAG compliant websites so that people with all types of di
 
 ## When we do this
 
-- We always produce work that is accessible to people of all abilities, regardless of client. However, we recognize that the level of accessibility compliance and prioritization can be influenced by budgetary and contractual implications.
+- We strive produce work that is accessible to people of all abilities, regardless of client. However, we recognize that the level of accessibility compliance and prioritization can be influenced by budgetary and contractual implications.
```

Comment:
Agreed, but you should change that to *strive _to_*.

Reply:
Good catch. So do I revise the simple patch or just wait for the reviewer to modify it?

Reply:
@mgifford I think you can add a commit to the PR branch to address this (anyone on the team can review most handbook changes, except for policy/legal stuff)